### PR TITLE
change tezos api provider

### PR DESCRIPTION
**Pull request by Pantani**
# Headline
TzScan is dead and is not working anymore

## Solution
Change API provider to DunScan instead TzScan

### CODE REVIEW

**Comment by kolya182** [comment]
No transactions returned for https://tezos.id/tz1bJnic8fHe7ZhFHWZ3degx3d4yJA13bafK

**Comment by Pantani** [comment]
> No transactions returned for https://tezos.id/tz1bJnic8fHe7ZhFHWZ3degx3d4yJA13bafK

weird

This is doesn't show anything
https://api6.dunscan.io/v3/operations/tz1bJnic8fHe7ZhFHWZ3degx3d4yJA13bafK?type=Transaction

this show only one transaction:
https://api6.dunscan.io/v3/operations/tz1WCd2jm4uSt4vntk4vSuUWoZQGhLcDuR9q?type=Transaction


Let me see what's happening

File: config.yml [hunk 1]
```diff
@@ -55,8 +55,8 @@ kin:
 
 # [XTZ] Tezos: https://tezos.com
 tezos:
-  api: https://api1.tzscan.io/v3
-  rpc: https://mainnet-node.tzscan.io
+  api: https://api6.dunscan.io
+  rpc: https://mainnet-node.dunscan.io
```
Comment (vikmeup) [inline config.yml]: dunscan is a different blockchain no?

Comment (dpereskokov) [reply config.yml]: If I got it right they now support Tezos as well
image

**Pull request merged**